.gitignore

```diff
@@ -1,4 +1,8 @@
 node_modules
 .vscode
-/.idea/
-*.DS_Store
\ No newline at end of file
+*.idea/
+*.DS_Store
+*.log
+examples/cordova-sample/MyApp/node_modules
+examples/cordova-sample/MyApp/platforms
+examples/cordova-sample/MyApp/plugins
```

Comment:
it looks like the plugins folder is actually getting added?

Reply:
good catch, I added before deciding to ignore, I'll kill it from the repo